specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -0,0 +1,164 @@
+
+# VEP-1243: VDK Meta jobs
+
+* **Author(s):** Yoan Salambashev (ysalambashev@vmware.com)
+* **Status:** draft
+
+- [Summary](#summary)
+- [Glossary](#glossary)
+- [Motivation](#motivation)
+- [Requirements and goals](#requirements-and-goals)
+- [High-level design](#high-level-design)
+- [API Design](#api-design)
+- [Detailed design](#detailed-design)
+- [Implementation stories](#implementation-stories)
+- [Alternatives](#alternatives)
+
+## Summary
+
+<!--
+Short summary of the proposal. It will be used as user-focused
+documentation such as release notes or a (customer facing) development roadmap.
+The tone and content of the `Summary` section should be
+useful for a wide audience.
+-->
+
+The Meta Job is a new feature in VDK that enables users to manage job dependencies in their data processing workflows.
+It allows a Data Job to trigger other Data Jobs only after they have successfully completed,
+resulting in more accurate and consistent data output. An integration with Apache Airflow is available,
+but it requires users to manage their own instance or pay for an externally managed service.
+Meta Jobs provide a more lightweight alternative that simplifies the process and reduces cost.
+
+## Glossary
+<!--
+Optional section which defines terms and abbreviations used in the rest of the document.
+-->
+
+## Motivation
+<!--
+It tells **why** do we need X?
+Describe why the change is important and the benefits to users.
+Explain the user problem that need to be solved.
+-->
+
+Data teams are constantly looking for ways to optimize and streamline their data processing workflows.
+One critical aspect of these workflows is managing job dependencies, which ensures that Data Jobs are executed
+in the appropriate order, leading to more accurate and consistent data output. Until now, we addressed this problem
+by integrating the external tool Apache Airflow. However, this approach comes with its downsides: teams would
+have to manage their own instance of Airflow, or pay for an externally managed service.
+
+To address this problem, VDK is introducing the Meta Jobs plugin. Meta Jobs are providing users with the ability
+to manage job dependencies natively within the framework. They are regular Data Jobs that invoke other Data Jobs using
+the Control Service Execution API. This new feature gives the chance to the user to easily orchestrate the workflow
+the way they want it, without the need to think about anything during the execution. The most important problem
+that Meta Jobs solve is that a certain Data Job will only be triggered after the successful completion
+(or other specified criteria) of one or more other Data Jobs.
+
+By introducing Meta Jobs, our data engineering framework empowers data teams to manage job dependencies in an elegant,
```

Comment:
"Elegant, quick and easy fashion" are subjective and qualitative terms. They are usually discouraged in technical writing like VEP because they cannot convey a precise and measurable meaning. Different people can have different interpretations of what is elegant, quick or easy, which can lead to confusion and miscommunication. Instead, VEP should use objective and quantitative terms that describe the features and benefits of the product in a precise and measurable way.

It would be helpful to provide more specific examples of how it can improve data processing workflows. For example, how does it make it easier for teams to manage dependencies? How does it save time and resources?

Reply:
Removed the aforementioned vague terms. Added some more info.